src/nss/on_pc_item_equ.nss

```diff
@@ -59,6 +59,7 @@ void main()
     object oPC   = GetPCItemLastEquippedBy();
 
     // Boomerang weapons should suppress "you are running out of ammo" messages
+    /* we will suppress this feedback permanently
```

Comment:
While keeping the feedback for mundane stuff would be nice, the simple on-equip thing I wrote before won't cut it once you have arrows as well as throwing.
Eg equipping magical arrows while wielding mundane thrown items would still suppress your notifications. Would have to check for launcher as well...